src/inputs/internal/ComboBoxInput.tsx

```diff
@@ -69,9 +84,19 @@ export function ComboBoxInput<O, V extends Value>(props: ComboBoxInputProps<O, V
   const showFieldDecoration =
     (!isMultiSelect || (isMultiSelect && !isFocused)) && fieldDecoration && selectedOptions.length === 1;
 
+  // To Discuss: Do we want to make `multiline = true` the default behavior?
+  // If we do, we can infer whether we allow wrapping or not based on the PresentationProp. This will currently only be set for "rowHeight: 'fixed'" GridTable styles.
+  // Or, we can keep our existing default behavior, and allow the user to set `multiline = true` if they want to allow wrapping.
+  // If we change the default then we could have unexpected text wrapping in places like Filters.
+  const multilineProps = wrap ? { textAreaMinHeight: 0, multiline: true } : {};
```

Comment:
Question: Should this `multiline: true` use the `multiline` local variable? Not sure...

Reply:
`multiline` should always be `true` if we ever way the text to wrap. There are two conditions in which we want the text to wrap; 1. The caller defined `multiline: true`, or 2. `PresentationContext.wrap` is set to `true`.
In this case it could be that `multiline` is `undefined`, but the PresentationContext property is defined as `true`, so we'd want to still set `multiline: true`.